canonical-websites/www.ubuntu.com
# Feature/273 legal pages

## Done

All legal pages updates

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- Check updates from [this epic](https://github.com/ubuntudesign/web-squad/issues/392) have been made
